test/cpp/jit/test_index_select.cpp

```diff
@@ -0,0 +1,722 @@
+#if defined(USE_CUDA)
```

Comment:
Please move this to `csrc/jit/codegen/cuda/test`.

Reply:
done. 
see backend PR: https://github.com/azazhu/pytorch/tree/fw/idx_sel_backend